docs/src/modules/ROOT/pages/using-timefold-solver/modeling-planning-problems.adoc

```diff
@@ -2840,12 +2840,162 @@ def generate_demo_data(demo_data: DemoData) -> Timetable:
 Usually, most of this data comes from your data layer,
 and your solution implementation just aggregates that data and creates the uninitialized planning entity instances for the solver to plan.
 
-[#planningModelInheritance]
-== Inheritance for Planning Entities
+[#planningModelSolutionInheritance]
```

Comment:
The inheritance changes deserve a note in the upgrade section.
No need to go into detail - just say that we formalized inheritance rules for solutions and entities, and that detailed specification can be found in these two sections. (With links to them.)